deploy/charts/trust-manager/values.yaml

```diff
@@ -1,146 +1,93 @@
-    port: 6060
-    # -- Path on which to expose trust-manager HTTP readiness probe using default network interface.
-    path: "/readyz"
-
-  trust:
-    # -- Namespace used as trust source. Note that the namespace _must_ exist
-    # before installing trust-manager.
-    namespace: cert-manager
-
-  webhook:
-    # -- Host that the webhook listens on.
-    host: 0.0.0.0
-    # -- Port that the webhook listens on.
-    port: 6443
-    # -- Timeout of webhook HTTP request.
-    timeoutSeconds: 5
-    # -- Type of Kubernetes Service used by the Webhook
-    service:
-      type: ClusterIP
-
-    tls:
-      approverPolicy:
-        # -- Whether to create an approver-policy CertificateRequestPolicy allowing auto-approval of the trust-manager webhook certificate. If you have approver-policy installed, you almost certainly want to enable this.
-        enabled: false
-
-        # -- Namespace in which cert-manager was installed. Only used if app.webhook.tls.approverPolicy.enabled is true
-        certManagerNamespace: "cert-manager"
-
-        # -- Name of cert-manager's ServiceAccount. Only used if app.webhook.tls.approverPolicy.enabled is true
-        certManagerServiceAccount: "cert-manager"
-
-    # -- Specifies if the app should be started in hostNetwork mode. Required for use in some managed kubernetes clusters (such as AWS EKS) with custom CNI.
-    hostNetwork: false
-
-  securityContext:
-    # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
-    seccompProfileEnabled: true
-
-  # -- Pod labels to add to trust-manager pods.
-  podLabels: {}
-
-  # -- Pod annotations to add to trust-manager pods.
-  podAnnotations: {}
-
 secretTargets:
-  # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target.
+  # If set to true, enable writing trust bundles to Kubernetes Secrets as a target.
   # trust-manager can only write to secrets which are explicitly allowed via either authorizedSecrets or authorizedSecretsAll.
   # NOTE: Enabling secret targets will grant trust-manager read access to all secrets in the cluster.
   enabled: false
-  # -- If set to true, grant read/write permission to all secrets across the cluster. Use with caution!
+  # If set to true, grant read/write permission to all secrets across the cluster. Use with caution!
   # If set, ignores the authorizedSecrets list.
   authorizedSecretsAll: false
-  # -- A list of secret names which trust-manager will be permitted to read and write across all namespaces.
+  # A list of secret names which trust-manager will be permitted to read and write across all namespaces.
   # These will be the only allowable Secrets that can be used as targets. If the list is empty (and authorizedSecretsAll is false),
   # trust-manager will not be able to write to secrets and will only be able to read secrets in the trust namespace for use as sources.
   authorizedSecrets: []
 
 resources: {}
```

Comment:
This field seems to be empty in the readme:
<img width="591" alt="image" src="https://github.com/cert-manager/trust-manager/assets/42113979/c6bb2389-d151-42ed-b1a2-3fa3ee14001d">